tests/playwright/fixtures/BRL.spec.mjs

```diff
@@ -24,7 +24,7 @@ for (const environment of environments) {
 
       await checkoutPage.goToCheckoutPageWithFullCart(regionsEnum.BR);
       await checkoutPage.setShopperDetails(shopperData.BR);
-      if (environment.name.indexOf("v6") === -1) {
+      if (environment.name.indexOf('v5') !== -1) {
```

Comment:
what is the logic behind this?
because previous logic was to set the email if the environment is not v6.
no with this logic it will set email also for v6 and v7.
shouldn't it stay only for v5?

Reply:
earlier it was setting email if not v6. which would have set it for v5 and v7. Now it checks for v5 and sets email only for v5.